README.md

```diff
@@ -1,23 +1,22 @@
 # MODULE_NAME Service
+*Replace the repository name for each:*
 
 ![Rust Checks](https://github.com/arrow-air/svc-template-rust/actions/workflows/rust_ci.yml/badge.svg?branch=main)
 ![Python Flake8](https://github.com/arrow-air/svc-template-rust/actions/workflows/python_ci.yml/badge.svg?branch=main)
 ![CodeClimate Coverage](https://img.shields.io/codeclimate/coverage/arrow-air/svc-template-rust)
-
-FIXME: Replace the repository name for each:
+![Arrow DAO Discord](https://img.shields.io/discord/853833144037277726?style=plastic)
 
 ## :telescope: Overview
-
-*FIXME: This is a high level description of this module.*
+*This is a high level description of this module.*
 
 ## :scroll: Documentation
-
+The following documents are relevant to this service:
 - [Concept of Operations](FIXME)
 - [Requirements & User Stories](FIXME)
 - [SDD](./docs/sdd.md)
 
 ## :busts_in_silhouette: Arrow DAO
-
+Learn about us:
```

Comment:
Learn more about us:

Reply:
Fixed in https://github.com/Arrow-air/svc-template-rust/pull/3/commits/003d7fde107fde2416c259c00b406c505e6093ab